### PR TITLE
Added style props to Alert component

### PR DESCRIPTION
**Pull request by jorgv**
Added text and border props to Alert component to make it more customizable using Vuetify native props

### CODE REVIEW

File: src/components/Alert.vue [hunk 1]
```diff
@@ -6,6 +6,8 @@
     :type="color"
     :outlined="outlined"
     :prominent="prominent"
+    :text="text"
```
Comment (yariksav) [inline src/components/Alert.vue:9]: Text already used for the message, look at 14 row, for this case i suppouse we need to use another parameter, for example flat, as it was beforein vuetify1

**Comment by yariksav** [comment]
Hi, @jorgv, thank you for PR, please see comment about "text"

File: src/components/Alert.vue (hunk 1 above)
Comment (jorgv) [reply src/components/Alert.vue:9]: Oh sorry I totally miss it, yes, flat sounds good I will edit it.

**Comment by jorgv** [comment]
@yariksav  thanks for the review, I already fixed it.

**Pull request merged**